tests/js/server/aql/aql-index-hints.js

```diff
@@ -421,7 +421,38 @@ function ahuacatlSkiplistOverlappingTestSuite () {
       assertEqual(usedIndexes[0][0], alternateSortingIndex);
     },
 
+    testUpsertWithIndexHint: function () {
+      const prefix = " UPSERT { a: 1234 } INSERT { a: 1234 } UPDATE {} IN " + cn + " OPTIONS ";
+      const indexHints = [
+        [{}, ["hash_a", "hash_a_b", "skip_a", "skip_a_b"]],
```

Comment:
hm, without MMFiles, these are merely names - @jsteemann any plan when we will drop these aliases?

Reply:
these names were used for preserving the behavior of the rest of the tests, all the tests in this file use the same collection and indexes, so I used the same ones for leaving it more uniform. 
It doesn't compromise the functionality of this specific test, which was to see if the indexes are being used actually as index hint